pkg/ingester/ingester_test.go

```diff
@@ -2631,11 +2643,13 @@ func TestIngester_QueryStream(t *testing.T) {
 					require.NoError(t, err)
 					assert.Greater(t, receivedSeries, 0)
 				}
+
+				expectedNumSeries = numSeries // Querying native histograms is not supported when query sharding is active.
```

Comment:
This test is testing ingester shards, not "query sharding" as in the query frontend . So I think the fact that you get less number of series is due to the filter not matching the new metric names - which we should not need , see my earlier comment.

```suggestion
```